src/fctGraph/Utils/mutators/addOutputFctForOutSlotWhichHasNone.js

```diff
@@ -37,11 +37,11 @@ function safeAddOutputFctAndConnect(fctGraph, outSlot, outputFct) {
   return addConnectionResponse
 }
 
-function addOutputFctForOutSlotWhichHasNone(fctGraph, outSlot, subTypeClass, opts) {
+function addOutputFctForOutSlotWhichHasNone(fctGraph, outSlot, SubTypeClass, opts) {
   assertPrerequisites(fctGraph, outSlot)
 
-  if ("customData" in opts) {
-    console.warn('DEPRECATION WARNING: "customData" key is deprecated. Please use "fctData" instead')
```

Comment:
does firmware know about this?

Reply:
I will tell make sure to tell them 

when working with slobodan we addressed the warnings he was getting - which he was previously ignoring thinking it was doing work that it was not. I think we must blow up here because if it silently passes they are under the wrong impression re: what just happened.